ui/src/helpers/api.js

```diff
@@ -45,9 +49,27 @@ export default class Api {
     }
 
     validateQuery = async (query) => {
+        if (this.source) {
+            this.source.cancel();
+        }
+        this.source = CancelToken.source();
+
         const form = new FormData();
         form.append('query', query)
-        const response = await this.client.post(`/query/validate`, form);
+        const response = await this.client.post(`/query/validate`, form, {
+            cancelToken: this.source.token
+        }).catch(function (thrown) {
+            if (axios.isCancel(thrown)) {
+                console.log('Validate canceled');
+            } else {
+                console.log('Validate error', thrown.message);
+            }
```

Comment:
```suggestion
            if (!axios.isCancel(thrown)) {
                console.error('Validate error', thrown.message);
            }
```

Reply:
@mertyildiran  what do you think about this?
I don't want to potentially spam when everything is good

Reply:
@gadotroee yeah that makes sense. Done with https://github.com/up9inc/mizu/commit/3016cd5bd3fa6dbad08fc239bd3706dc6e9b8206